src/db/service/Unfollow.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable valid-jsdoc */
+import models from '../models';
+
+const { FollowSummary, } = models;
+
+/**
+ * Follow Service
+ */
+class Unfollow {
+  /**
+   * @static
+   * @param {number} req
+   * @memberof UnFollow
+   * @returns Updated details | false
+   */
+  static async unfollower(id) {
+    const findFollowingSummary = await FollowSummary.findOne({
+      where: {
+        userId: id,
+      }
+    });
+    if (findFollowingSummary) {
+      const following = findFollowingSummary.following - 1;
+      findFollowingSummary.following = following;
+      findFollowingSummary.save();
```

Comment:
can you explain this line?

Reply:
This line simply updates the current number of people a user follows in the summary table. So, the `save` is an accessor function on `sequelize` that immediately updates the specified table.

Take note of the `following` variable passed into the `findFollowingSummary` object. The variable holds the number of artists a user currently follows.